README.md

```diff
@@ -57,6 +57,52 @@ export class OtherModule {
 }
 ```
 
+### Including only what you need
+For production scenarios, you will likely only want to include the parts of ng-bootstrap that you are using in your application.
+To do so, you can import only the specific modules that you need. For example:
+
+ ```js
+ import { NgbModalModule } from '@ng-bootstrap/ng-bootstrap/modal/modal.module';
```

Comment:
Like I mentioned above, I personally don't have this problem when importing the modules. But for importing things like `NgbModal` while in lazily loaded modules, I do have to use the deep imports, otherwise the rest of ng-bootstrap gets pulled in.